collaboration-service/index.js

```diff
@@ -0,0 +1,56 @@
+import express from "express";
+import cors from "cors";
+import { createServer } from "http";
+import "dotenv/config";
+import { Server } from "socket.io";
+
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(cors());
+app.options("*", cors());
+
+const httpServer = createServer(app);
+const PORT = process.env.PORT || 8003;
+
+httpServer.listen(PORT, () => {
+  console.log(`Listening on port ${PORT}`);
+});
+
+// Also tested via Postman
+const io = new Server(httpServer, {
+  cors: {
+    origin: true,
```

Comment:
Would it be better to create socket server first before listening?

Reply:
Yep changed that